### ISSUE TITLE
Provide endpoint to delete registered query in data audit

### ISSUE THREAD

**Issue by martinweiler** [apache/incubator-kie-issues#857]
Following up on #834 an endpoint should be provided that allows to delete a query that has been registered previously.

**Comment by elguardian** [comment]
duplicated.
https://github.com/apache/incubator-kie-issues/issues/761

**Comment by martinweiler** [comment]
I don't think this is a duplication of #761. The use case for the other issue is "delete audit data older than X" whereas this issue is about providing the user with the ability to delete a specific query (eg. one that has been registered by mistake or one that is no longer needed).
